python/hail/methods/impex.py

```diff
@@ -1714,6 +1714,59 @@ def import_vcf(path,
     return MatrixTable(jmt)
 
 
+@typecheck(metadata_file=str,
+           call_fields=oneof(str, sequenceof(str)),
+           reference_genome=nullable(reference_genome_type),
+           entry_field_array_elements_required=bool,
+           tar=nullable(str))
```

Comment:
Is it easy for you to add `contig_recoding` like `import_vcf`? Otherwise, I'll add it in a separate PR once this goes in.

Also, the `entry_field_array_elements_required` is slightly different from `import_vcf`. Is there a way to unify them?